src/include/connection.h

```diff
@@ -335,8 +335,8 @@ extern const WT_NAME_FLAG __wt_stress_types[];
 
 /*
  * Access the array of all sessions. This field uses the Slotted Array pattern to managed shared
- * accesses, if you are looking to walk all sessions please consider using the existing session walk
- * logic. FIXME-WT-10946 - Add link to Slotted Array docs.
+ * accesses, if you need to walk all sessions please call __wt_session_array_walk. For more details
```

Comment:
We should still mention this is the slotted array pattern in the comment.

Reply:
It does lol.

Reply:
So it does. I got fixated on the red diff